src/sfProject.ts

```diff
@@ -544,6 +604,17 @@ export class SfProject {
     return packageDir ? packageDir.name : undefined;
   }
 
+  /**
+   * Returns the package directory that has the given property whose value equals the given value.
+   *
+   * @param property
+   * @param value
+   */
+  public getPackageDirectoryWithProperty(property: string, value: string): Optional<NamedPackageDir> {
```

Comment:
Is this a common usecase?  This is essentially a filter.  I.e., `getPackageDirectories().filter(pd => pd.prop === value)` where the filter returns a single match.

Reply:
Right now the function I wanted to replace from pkg utils is used twice, both are looking up which package dir is the default. I will remove this for now and we can  remove refs to `getConfigPackageDirectory` in pkg utils with calls to project.getDefaultPackage